packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -464,19 +465,78 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
 
         // TODO: This snapshot will return deltas, which we currently aren't using. We need to enable this flag to go down the "optimized"
         // snapshot code path. We should leverage the fact that these deltas are returned to speed up the deltas fetch.
-        const { headers, url } = getUrlAndHeadersWithAuth(`${this.snapshotUrl}/trees/latest${snapshotOptions}`, storageToken);
+        const { headers, url, isOptionsCall } = getUrlAndHeadersAndOptionsWithAuth(`${this.snapshotUrl}/trees/latest${snapshotOptions}`, storageToken);
 
         // This event measures only successful cases of getLatest call (no tokens, no retries).
         const { snapshot, canCache } = await PerformanceEvent.timedExecAsync(this.logger, { eventName: "TreesLatest" }, async (event) => {
             const response = await fetchHelper<IOdspSnapshot>(url, { headers });
             const content = response.content;
+            let isResourceDataAvailable = false;
+            let dnstime; // domainLookupEnd - domainLookupStart
```

Comment:
>dnstime [](start = 16, length = 7)

These are all numbers, please add typing.
But if you initialize these all to -1 upfront, the typescript compiler will be able to infer these as numbers.

Reply:
resolved this.